src/js/actions/layers.js

```diff
@@ -881,11 +881,8 @@ define(function (require, exports) {
             return Promise.resolve();
         }
 
-        if (modifier !== undefined && layerSpec.size > 1) {
-            return Promise.reject(new Error("Select with modifier requires a single layer"));
-        }
-
-        var selected = document.layers.selected,
+        var setPivot = modifier === undefined || layerSpec.size === 1,
```

Comment:
Sorry, I meant that this line should just be `var setPivot = layerSpec.size === 1` since the undefined case does not rely on setPivot